aea/skills/base.py

```diff
@@ -389,18 +389,70 @@ def parse_module(
 class Task(SkillComponent):
     """This class implements an abstract task."""
 
-    def __init__(self, *args, **kwargs):
+    def __init__(
```

Comment:
better having `name` and `skill_context` explicit if possible - to be extended to the other skill component classes eventually.